.github/renovate.json5

```diff
@@ -38,6 +38,13 @@
       "fileMatch": ["^hack\/.+\\.sh$"],
       "matchStrings": ["--image[=| ][\"|']?(?<depName>.*?):(?<currentValue>.*?)[\"|']?\\s"],
       "datasourceTemplate": "docker"
+    },
+    {
+      // Generic detection of container images in images.yaml.
+      "customType": "regex",
+      "fileMatch": ["^imagevector\/images.yaml$"],
+      "matchStrings": ["\\s+repository:\\s+(?<depName>.*?)\\n\\s+tag:\\s+[\"]?(?<currentValue>.*?)[\"]?\\n"],
+      "datasourceTemplate": "docker"
```

Comment:
There is a bug (https://github.com/renovatebot/renovate/discussions/27505) in `renovate` which prevents it from accessing container registries on Google Cloud when it is using GCP VMs.
[imagevector/images.yaml](https://github.com/gardener/gardener/blob/master/imagevector/images.yaml) contains some references to `gcr.io` and `registry.k8s.io` might use a Google backend too when it is accessed from an GCP VM ([ref](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/#why-isn-t-there-a-stable-list-of-domains-ips-why-can-t-i-restrict-image-pulls)). This would limit the update sources which are working.

I just randomly checked several dependencies and most of them create Github releases too. Thus, you could add a similar additional rule which uses `github-releases` as `datasourceTemplate` and get the dependency name from `sourceRepository` in images.yaml.
WDYT?

Reply:
Valid point. Until the bug is fixed we can use `github-releases` in parallel. Afterwards, we may decide for just one of the solutions or simply use both for high availability ;-) .

Addressed with https://github.com/gardener/gardener/pull/9370/commits/1ed8aa376338a17c37126c56edca2c34dda22ac0.

WDYT?

Reply:
looks good 🤖